README.md

```diff
@@ -9,8 +9,8 @@ E2e-runner coordinates the entire e2e run against a Workflow [chart](https://git
 
   1. Coordinating the leasing of a GKE k8s cluster via [k8s-claimer](https://github.com/deis/k8s-claimer),
   2. Cleaning up the leased cluster if need be (primarily deleting the `deis` namespace if still exists),
-  3. Setting up the local [helmc](https://github.com/helm/helm-classic) install on the leased cluster,
-  4. Generating and installing the Workflow and [Workflow-e2e](https://github.com/deis/charts/tree/master/workflow-dev-e2e) charts,
+  3. Setting up the local [helm](https://github.com/kubernetes/helm) install on the leased cluster (deleting an existing `tiller` deployment if necessary),
```

Comment:
@mboersma good point, it's a bit of a tradeoff.  However, as it's a close call, I think I do prefer the maintainability gain in all 1's... let's go with that this time 'round.